test/e2e_node/util.go

```diff
@@ -417,6 +417,8 @@ func runCommand(cmd ...string) (string, error) {
 func getCRIClient() (internalapi.RuntimeService, internalapi.ImageManagerService, error) {
 	// connection timeout for CRI service connection
 	const connectionTimeout = 2 * time.Minute
+	// image pull timeout for CRI service
+	const imagePullProgressDeadline = 60 * time.Minute
```

Comment:
Nit: 60 minutes seems like a really high value here. Is there any world in which we'd want to wait the full 60 minutes instead of just failing the tests and trying again?

Reply:
Because previous time is forever, I think set a longer time would be better. But let me change to 1 min which is default value of image pull progress deadline in docker.